src/app/admin/dlogin/dlogin.component.html

```diff
@@ -11,49 +11,49 @@ <h3 class="fw-bold mb-4">{{'SCHEMA' | translate}}</h3>
 
         <div class="d-flex flex-row row mb-3">
             <div class="card p-4 col-sm-2 mr-2">
-                <p class="p16 mb-1 font-600">{{'FACILITY_CENTER' | translate}}</p>
+                <p class="fs-16 mb-1 font-600">{{'FACILITY_CENTER' | translate}}</p>
                 <h2 class="font-600">0</h2>
-                <p class="p12 mb-1">{{'RECORD' | translate}}</p>
-                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">{{'VIEW_ADD' | translate}}</a>
+                <p class="fs-12 mb-1">{{'RECORD' | translate}}</p>
+                <a class="my-2 fs-14 btn bg-color-light-blue color-sea-green submit-button align-self-start">{{'VIEW_ADD' | translate}}</a>
             </div>
 
             <div class="card p-4 col-sm-2 mr-2">
-                <p class="p16 mb-1 font-600">{{'DOCTORS | translate'}}</p>
+                <p class="fs-16 mb-1 font-600">{{'DOCTORS | translate'}}</p>
                 <h2 class="font-600">0</h2>
-                <p class="p12 mb-1">{{'RECORD' | translate}}</p>
-                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">{{'VIEW_ADD' | translate}}</a>
+                <p class="fs-12 mb-1">{{'RECORD' | translate}}</p>
+                <a class="my-2 fs-14 btn bg-color-light-blue color-sea-green submit-button align-self-start">{{'VIEW_ADD' | translate}}</a>
```

Comment:
@chaitrali-r dont give class name based on color code. Color code will be change for differnet theme.
- bg-color-light-blue 
- color-sea-green